[BLOCKED] Release/1.0.0 (#3498)
aiidateam/aiida-core: AUTHORS.txt

> This file lists the authors of the AiiDA code (i.e. the members of the AiiDA team)

Reviewer: ```suggestion
This file lists the authors of the AiiDA code in alphabetical order, including both the members of the AiiDA team at EPFL as well as external contributors.
```

> as well as the known contributors to the code.

Reviewer: ```suggestion
```

> The current members of the AiiDA team are, in alphabetical order:

Reviewer: ```suggestion
The AiiDA team at EPFL
```

* Fernando Gargiulo (2016-2017)
* Andrius Merkys (2015-2017)
* Andrea Cepellotti (2012-2016)
* Boris Kozinsky (2012-2016)
* Riccardo Sabatini (2013-2015)

Contributors
############
We would like to thank the following people for their major contibutions:

* Dominik Gresch (ETHZ): for the caching mechanism and contributions to the workflow engine
* Tiziano Müller (ETHZ, Switzerland): for the first implementation of
  the new plugin system (developed in 2017) and making the code python 3 compatibl

And the following people for general improvements to the code, fixing bugs,
corrections and improvements to the documentation and useful suggestions:

* Oscar Arbelaez
* Michael Atambo
* Valentin Bersier
* Marco Borelli
* Jocelyn Boullier
* Jens Bröder
* Ivano E. Castelli
* Keija Cui
* Vladimir Dikan
* Marco Dorigo
* Y.-W. Fang
* Marco Gibertini
* Daniel Hollas
* Eric Hontz
* Jianxing Huang
* Espen Flage Larsen
* Ian Lee
* Daniel Marchand
* Antimo Marrazzo
* Simon Pintarelli
* Gianluca Prandini
* Philipp Rüßmann
* Philippe Schwaller
* Ole Schütt
* Chris Sewell
* Andreas Stamminger
* Atsushi Togo
* Daniele Tomerini
* Nicola Varini
* Jason Yu
* Bonan Zhu
* Mario Zic

---

> Please, don't hesitate to contact the AiiDA team if you see that information is wrong or missing in this file.

Reviewer: ```suggestion
Please, don't hesitate to contact the AiiDA team if you notice any missing or incorrect information.
```
